exercises/diamond/DiamondTest.cs

```diff
@@ -1,110 +1,131 @@
-﻿using System;
-using System.Collections.Generic;
+﻿using Xunit;
+using FsCheck.Xunit;
+using FsCheck;
+using System;
 using System.Linq;
-using Xunit;
 
-public class DiamondTest
+namespace Diamond
```

Comment:
Could you remove the namespace? We don't use namespaces in any of the exercises to make things as simple and clean as possible.